cmd/controller/app/controller.go

```diff
@@ -326,15 +326,15 @@ func startLeaderElection(ctx context.Context, opts *options.ControllerOptions, l
 		return fmt.Errorf("error getting hostname: %v", err)
 	}
 
-	// Set up Multilock for leader election. This Multilock is here for the
-	// transitionary period from configmaps to leases see
-	// https://github.com/kubernetes-sigs/controller-runtime/pull/1144#discussion_r480173688
 	lockName := "cert-manager-controller"
 	lc := resourcelock.ResourceLockConfig{
 		Identity:      id + "-external-cert-manager-controller",
 		EventRecorder: recorder,
 	}
-	ml, err := resourcelock.New(resourcelock.ConfigMapsLeasesResourceLock,
+
+	// We only support leases for leader election. Previously we supported ConfigMap & Lease objects for leader
+	// election.
+	ml, err := resourcelock.New(resourcelock.LeasesResourceLock,
```

Comment:
We could also explicitly switch to `Lease`s in cainjector and update the it's RBAC https://github.com/cert-manager/cert-manager/blob/master/cmd/cainjector/app/start.go#L137

Reply:
Implemented